chore: add github issues templates (#740)
snyk/kubernetes-monitor: .github/ISSUE_TEMPLATE/BUG_REPORT.md

---
name: "🐛 Bug report"
about: Submit a bug
title: "[🐛]"
labels: "🐛  bug"
assignees: ''

> ---

Reviewer: I don't quite understand this table, how is it meant to be used?

Author: Apparently, it transforms into this table with requirements to the Issue look
https://github.com/snyk/snyk/blob/master/.github/ISSUE_TEMPLATE/bug-report.md

- `kubernetes-monitor version [e.g. v1.53.0]` 
- `Cloud runtime [e.g. GKE, EKS]`

### Expected behaviour

> Please share _expected_ behaviour.

Reviewer: ```suggestion
Please share the _expected_ behaviour.
```

### Actual behaviour

> Please share _problematic_ behaviour you are seeing.

Reviewer: ```suggestion
Please share the _problematic_ behaviour you are seeing.
```

### Steps to reproduce

> Please share _minimal_ steps needed to reproduce your issue. Ideally

Reviewer: ```suggestion
Please share the _minimal_ steps needed to reproduce your issue. Ideally
```

> a paired down manifest / project to showcase the problem that can also

Reviewer: ```suggestion
a container image or a YAML file to showcase the problem that can also
```
